### DISCUSSION TITLE
Support for Waypoint Adveraging

### DISCUSSION THREAD

**Discussion by TheBoshie** [opengisch/QField#2566] [Ideas]
Hi! I love the Qfield program you are making!

as I hope to replace my Garmin GPS eventually with Qfield when I am working outside, I was curious if you have looked at waypoint averaging as a feature?

In my line of work I need to get sub-meter measurements in specific situations, and typically we have found using a phones ~4 meter accuracy with waypoint averaging would work.

**Comment by ZsanettMed** [comment]
Hi @TheBoshie!

Meanwhile, I'm pleased to confirm that QField does support averaged positioning functionality. For more information, please visit here: https://docs.qfield.org/how-to/gnss/?h=aver#averaged-positioning-functionality.

Best, 
Zsanett
